### PR TITLE
Update the feature-policy IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

File: interfaces/feature-policy.idl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+partial interface HTMLIFrameElement {
```
Comment (foolip) [inline interfaces/feature-policy.idl]: Wow, this is the least amount of IDL I've seen in any spec:
https://wicg.github.io/feature-policy/#integrations

**Review by foolip — CHANGES_REQUESTED** [review]
Needs a test to use it, and I should go home.

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-08 02:08:10
Finished: 2018-03-08 02:15:23





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25511)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/350509931)

**Comment by lukebjerring** [comment]
Added a test, PTAL.

File: feature-policy/interfaces.any.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+"use strict";
+
+if (self.importScripts) {
```
Comment (foolip) [inline feature-policy/interfaces.any.js]: This bits ought to not be needed, sent https://github.com/w3c/web-platform-tests/pull/10033 to confirm.

**Review by foolip — APPROVED** [review]
LGTM % the extra bits

**Pull request merged**